src/components/AvatarGroup/AvatarGroupCounterTooltipContentVirtualizedList.tsx

```diff
@@ -12,7 +12,7 @@ const LIST_OPTIONS = Object.freeze({
 });
 
 export type AvatarItem = {
-  value: AvatarProps & { tooltipContent: string | ReactElement };
+  value: AvatarProps & { tooltipContent: string | ReactElement | ReactElement[] };
```

Comment:
Because it was throwing error because the tooltip receive a list of people info to display in the tooltip content instead of one react element